assets/scss/base/_table.scss

```diff
@@ -277,3 +277,121 @@ Styleguide Table.Marker
   height: 10px;
   border-radius: 50%;
 }
+
+/*
+Responsive
+
+Responsive Table
+
+Markup:
+<table>
+  <thead>
+    <tr class="table__row--desktop-headings">
+      <th>Heading 1</th>
+      <th>Heading 2</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr class="table__row--responsive">
+      <td class="table__detail--responsive" data-heading="Heading 1">
+        <p class="table__cell-content--responsive">Row 1 Detail 1<p>
+      </td>
+      <td class="table__detail--responsive" data-heading="Heading 2">
+        <p class="table__cell-content--responsive">Row 1 Detail 2</p>
+      </td>
+    </tr>
+    <tr class="table__row--responsive">
+      <td class="table__detail--responsive" data-heading="Heading 1">
+        <p class="table__cell-content--responsive">Row 2 Detail 1</p>
+      </td>
+      <td class="table__detail--responsive" data-heading="Heading 2">
+        <p class="table__cell-content--responsive">Row 2 Detail 2</p>
+      </td>
+    </tr>
+    <tr class="table__row--responsive">
+      <td class="table__detail--responsive" data-heading="Really quite a long heading for column 1">
+        <p class="table__cell-content--responsive">Row 3 Detail 1</p>
+      </td>
+      <td class="table__detail--responsive" data-heading="Really quite a long heading for column 2">
+        <p class="table__cell-content--responsive">Row 3 Detail 2</p>
+      </td>
+    </tr>
+    <tr class="table__row--responsive">
+      <td class="table__detail--responsive" data-heading="Heading 1">
+        <p class="table__cell-content--responsive">Row 4 Detail 1 - has really quite a lot of content to include.</p>
```

Comment:
Hi, yeah, it was just to add enough content to the cells so that they exceed the width available and show what happens when text overflows on to multiple lines. Running locally seemed OK to me, but I've reworded - let me know what you think.